spec/lib/base/glooko_logging_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe 'GlookoLogging' do
+  let(:logger_class) { Class.new { include Base::GlookoLogging } }
+
+  it 'logs expected output with json data' do
+    test = expect do
```

Comment:
can we write this using lazy variables?

let(:test_log) do 
  logger_class.new.log('warn', 
                                      {fog: 'test'}, 
                                      __LINE__, 
                                      __method__,
                                       __FILE__) 
end

expect(test_log).to output()...

Reply:
Changed the code to fit the suggested comment